src/Agda2Hs/Compile/Function.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE OverloadedStrings, ViewPatterns, NamedFieldPuns #-}
+{-# LANGUAGE OverloadedStrings, ViewPatterns, NamedFieldPuns, BlockArguments #-}
```

Comment:
I don't think `BlockArguments` are used anywhere in this file, unless I missed it.